main.ts

```diff
@@ -1,14 +1,25 @@
 import "./deps/shims/register.ts"
-import { Command } from "./deps/cliffy.ts"
-
 import { bin } from "./cli/bin.ts"
 import { serve } from "./cli/serve.ts"
 import { sync } from "./cli/sync.ts"
+import { Command } from "./deps/cliffy.ts"
+import { detectVersion } from "./server/detectVersion.ts"
 
 await new Command()
   .name("capi")
+  .version(version)
   .description("Capi is a framework for crafting interactions with Substrate chains")
   .command("bin", bin)
   .command("sync", sync)
   .command("serve", serve)
   .parse(Deno.args)
+
+function version() {
+  try {
+    return detectVersion()
+  } catch (_e) {
+    return new TextDecoder().decode(
```

Comment:
This should only be done in deno, and if `import.meta.url` is a file url. I.e., it should be moved to `detectVersion`.